### DISCUSSION TITLE
The Dinosaur icon of Firefox

### DISCUSSION THREAD

**Discussion by nestoris** [nestoris/Win98SE#32] [Polls]
Don't you think a dinosaur-shaped Firefox icon is inappropriate? Some people have already asked me to change it to the old version of the fox that wrapped the globe, but I undeservedly ignored them. My wife and daughter are also asking me to remove this terrible red crocodile. I can change this icon individually on my computer, but maybe it still confuses someone? So, vote!

**Comment by nestoris** [comment]
In any case, there will be both versions and switch scripts (as is the case with the 98/2000 style of "My Computer")

This is Dinosaur:
![](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/apps/48/seamonkey.png)
This is another Firefox icon:
![](https://upload.wikimedia.org/wikipedia/commons/2/26/Mozilla_Firefox_logo_2004.svg)

**Comment by shamefulCake1** [comment]
There still is Seamonkey, which is Mozilla Suite in disguise. 

I use both Firefox and Seamonkey, and the dragon is firmly associated with the Suite, not the standalone browser, for me.

**Reply by ikkqqstudio** [reply]
Bro Is Typing Facts💯

**Reply by nestoris** [reply]
So be it. I will install **Fox on Globe** to a Firefox icon, and the **Dragon** (actually a Dinosaur) to a Seamonkey.
